_topic_map.yml

```diff
@@ -119,6 +119,23 @@ Topics:
   - Name: Autoscaling worker nodes
     File: rosa-autoscale-nodes
 ---
+Name: Monitoring user workloads
```

Comment:
I sent a message to Christian and am waiting to hear from him. But I'm going to change this title anyway because we use `user-defined projects` elsewhere in this Monitoring chapter for OSD and ROSA.